### ISSUE TITLE
Incorporate ontologies to improve search results

### ISSUE THREAD

**Issue by pkraker** [pkraker/Headstart#60]
As suggested by @brembs in #43: "How about using the various ontologies? For instance, gene ontology is widely used and should be quite up to date. I haven't checked it in a while, but there are numerous behavior-related terms in GO and 'operant' might be there along with related concepts."

For Pubmed integration, we could take a look at MeSH: https://www.nlm.nih.gov/mesh/

**Comment by nthmost** [comment]
Yeah, MeSH tags are a good for helping link papers together once you know what the user is asking for.  MedGen and the use of PubMed Advanced Query will be helpful in interpreting and disambiguating users' search terms.

*raises hand to help out*   :)

**Comment by pkraker** [comment]
@nthmost thanks for your input!

> _raises hand to help out :)_

awesome! will get back to you soon.
